File: src/main/java/com/ebuozturk/ecommerce/dto/UpdateCartRequest.java
package com.ebuozturk.ecommerce.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.Date;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class UpdateCartRequest {
    private Long id;
    private String name;
    private String no;
    private Date expiryDate;
    private String cvc;
    private Long userId;
}

File: src/main/java/com/ebuozturk/ecommerce/dto/cart/UpdateCartRequest.kt
package com.ebuozturk.ecommerce.dto.cart;

import java.util.*

data class UpdateCartRequest @JvmOverloads constructor(
    val id: Long,
    val name: String,
    val no: String,
    val expiryDate: Date,
    val cvc: String,
    val userId: Long,
) {
}